CONTRIBUTING.md

```diff
@@ -48,92 +52,142 @@ When you introduce a new whitelist comment, please don't forget to update the [w
 
 # Unit Testing
 
-TL;DR
+## Pre-requisites
+* WordPress-Coding-Standards
+* PHP CodeSniffer 2.9.x or 3.x
+* PHPUnit 4.x or 5.x
 
-If you have installed `phpcs` and the WordPress-Coding-Standards as [noted in the README](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards#how-to-use-this), then you can navigate to the directory where the `phpcs` repo is checked out and do:
+The WordPress Coding Standards use the PHP CodeSniffer native unit test suite for unit testing the sniffs.
 
-```sh
-composer install
-vendor/bin/phpunit --filter WordPress tests/AllTests.php
+Presuming you have installed PHP CodeSniffer and the WordPress-Coding-Standards as [noted in the README](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards#how-to-use-this), all you need now is `PHPUnit`.
+
+If you already have PHPUnit installed on your system: Congrats, you're all set.
+
+If not, you can navigate to the directory where the `PHP_CodeSniffer` repo is checked out and do `composer install` to install the `dev` dependencies.
+Alternatively, you can [install PHPUnit](https://phpunit.de/manual/5.7/en/installation.html) as a PHAR file.
+
+**Pro-tip**: PHPUnit is easiest to work with if you add the installation directory to your `PATH` environment variable.
```

Comment:
OK - I should clarify - it's not best practice according to my understanding of https://12factor.net/dependencies (or https://roots.io/twelve-factor-02-dependencies/ if you prefer).

For each of my projects, I include WPCS as a `require-dev`. I only use the global git checkout of PHPCS and WPCS for Editors which don't have project-specific settings.